sass/px-view-header.scss

```diff
@@ -29,10 +24,38 @@
 // Component
 .header {
   min-height: 4rem;
-  background-color: var(--px-view-header-background-color, white);
-  padding: 1rem;
+  background-color: var(--px-view-header-background-color, lightgray);
+  //flex stuff
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.headerText {
+  padding-left: $inuit-base-spacing-unit;
+  padding-right: $inuit-base-spacing-unit;
+  //flex stuff
+  flex: initial;
+  min-width: 0;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.truncate {
```

Comment:
I know we don't do this everywhere but I've been finding it help to prefix classes with the component they are in so that it is easier to see where classes are defined. maybe you could prefix your classes with `vh-`